.goreleaser.yaml

```diff
@@ -0,0 +1,52 @@
+# This is an example .goreleaser.yml file with some sensible defaults.
+# Make sure to check the documentation at https://goreleaser.com
+before:
+  hooks:
+    - make fmt
+    - make vet
+builds:
+  - id: 'cli'
+    env:
+      - CGO_ENABLED=0
+    goos:
+      - linux
+      #      - windows # coming soon?
+      - darwin
+    goarch:
+      - 'amd64'
+      - '386'
+      - 'arm64'
+    ignore:
+      - goos: 'darwin'
+        goarch: '386'
+      - goos: 'linux'
+        goarch: 'arm'
+        goarm: '7'
+      - goarm: 'mips64'
+      - gomips: 'hardfloat'
+      - goamd64: 'v4'
+    skip: false
+    no_unique_dist_dir: true
+    mod_timestamp: '{{ .CommitTimestamp }}'
+    binary: '{{ .ProjectName }}_{{ .Os }}_{{ .Arch }}_v{{ .Version }}'
+
+archives:
+  - id: 'cli'
+    name_template: '{{ .ProjectName }}_{{ .Version }}_{{ .Os }}_{{ .Arch }}'
```

Comment:
```suggestion
    name_template: '{{ .ProjectName }}_{{ .Os }}_{{ .Arch }}_{{ .Version }}'
```

Reply:
Shall we keep this same as the naming format for the binary?

Reply:
done